app/controllers/api/v1/schools_controller.rb

```diff
@@ -28,4 +28,12 @@ def show
       }, status: :not_found
     end
   end
+
+  # GET schools/search?partial=schoo
+  def search
+    if params['partial'].present?
```

Comment:
I don't think partial searches should be using another query param. Can you combine this with `=name`?

Reply:
Moved this to use the name parameter